docs/content/dev/notes.md

```diff
@@ -0,0 +1,50 @@
+# Notes
+
+Each note in HedgeDoc 2 contains the following information:
+
+- publicId (`b604x5885k9k01bq7tsmawvnp0`)
+- a list of aliases (`[hedgedoc-2, hedgedoc-next]`)
+- groupPermissions
+- userPermissions
+- viewCount (`0`)
+- owner
+- revisions
+- authorColors
+- historyEntries
+- description (`All you never wanted to know about notes`)
+- title (`Notes`)
+- tags (`[features, cool, update]`)
```

Comment:
Should a note also have a version field? For example to handle redirects from existing "root notes" e.g. `https://hedgedoc.example.com/mynote` to the `/n/` prefix?

Reply:
Interesting idea, @ErikMichelson also suggested that, maybe we should add this.